codeforamerica/vita-min
# FYST 239 Update AZ State Credits to AZ Subtractons

## Link to pivotal/JIRA issue
https://codeforamerica.atlassian.net/browse/FYST-239

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
az-state-credits page has been updated to az-subtractions page

## How to test?
Use heroku to set the time to intake time in session_toggles and step through the az flow. Use the df return for Tycho and check the az-subtractions page after prior last names. 

## Screenshots (for visual changes)
- Before
<img width="1002" alt="Screenshot 2024-07-31 at 10 13 37 AM" src="https://github.com/user-attachments/assets/0c7f98e1-031f-43ad-89e5-52b4d68563ba">
- After
<img width="992" alt="Screenshot 2024-07-31 at 10 13 18 AM" src="https://github.com/user-attachments/assets/1acef9a8-1c84-4632-bc07-a9eeee743d9a">
